relay-server/src/endpoints/minidump.rs

```diff
@@ -156,15 +155,15 @@ async fn handle(
 }
 
 pub fn route(config: &Config) -> MethodRouter<ServiceState> {
-    post(handle).route_layer(DefaultBodyLimit::max(config.max_attachments_size()))
+    // Set the single-attachment limit that applies only for raw minidumps. Multipart bypasses the
+    // limited body and applies its own limits.
+    post(handle).route_layer(DefaultBodyLimit::max(config.max_attachment_size()))
```

Comment:
This might be surprising behavior. `DefaultBodyLimit` applies by default for all extractors that `axum` defines, since they use `with_limited_body()` internally. Among others, this applies to the `Bytes` extractor, which is used in one of the branches of `handle`. 

In our case, we want two different limits:
1. For raw minidumps, use the _single_ attachment limit
2. For multipart, use the _multiple_ attachments limit. 

Our multipart extractor does that automatically, as it has access to our config, and that also bypasses `with_limited_body`. Hence, the limit here applies to just one of the two branches. 

I've been looking into ways to layer the limit directly onto the Bytes extractor, but haven't found a good way to do so.